pipewire/qubes-pw-module.c

```diff
@@ -303,8 +332,40 @@ struct impl {
 #endif
 
     atomic_uint_fast64_t reference_count;
+
+    const char *args;
 };
 
+int get_domid_from_props(struct pw_properties *props) {
+    size_t domid;
+    const char *peer_domain_prop = NULL;
+    if ((peer_domain_prop = pw_properties_get(props, QUBES_AUDIOVM_PW_KEY)) == NULL) {
+        return -EINVAL;
+    }
+    if (parse_number(peer_domain_prop, INT_MAX / 2, &domid, "domain ID")) {
+        pw_log_debug("Cannot parse domid");
+        return -errno;
+    }
+    return domid;
```

Comment:
Please document in the commit message that `QUBES_AUDIOVM_PW_KEY` is no longer set.